external/db_drivers/liblmdb/mdb.c

```diff
@@ -10707,9 +10707,11 @@ mdb_reader_list(MDB_env *env, MDB_msg_func *func, void *ctx)
 	for (i=0; i<rdrs; i++) {
 		if (mr[i].mr_pid) {
 			txnid_t	txnid = mr[i].mr_txnid;
-			sprintf(buf, txnid == (txnid_t)-1 ?
-				"%10d %"Z"x -\n" : "%10d %"Z"x %"Y"u\n",
-				(int)mr[i].mr_pid, (size_t)mr[i].mr_tid, txnid);
+			if (txnid == (txnid_t)-1) {
+				sprintf(buf, "%10d %"Z"x -\n", (int)mr[i].mr_pid, (size_t)mr[i].mr_tid);
+			} else {
+				sprintf(buf, "%10d %"Z"x %"Y"u\n", (int)mr[i].mr_pid, (size_t)mr[i].mr_tid, txnid);
+			}
 			if (first) {
 				first = 0;
 				rc = func("    pid     thread     txnid\n", ctx);
```

Comment:
Looks like this one will not be fixed upstream:
http://www.openldap.org/its/index.cgi/Incoming?id=8407
> The code is correct. Closing this ITS.
> 
> It is not the compiler's job to scold programmers about style.